poker_workflow.py

```diff
@@ -0,0 +1,35 @@
+import ast
+from temporalio import workflow
+from game_state import GameState
+from deck_utils import deal_cards, create_deck, shuffle_deck
+from hand_ranking_workflow import HandRankingWorkflow, HandRankingInput
+
+
+@workflow.defn
+class PokerWorkflow:
+    @workflow.run
+    async def run(self, seed: int) -> None:
+        deck = await shuffle_deck(create_deck(), seed)
+        game_state = GameState(deck=deck, players=[[], [], [], []])
+
+        for i in range(4):
+            game_state.players[i] = await deal_cards(game_state, 5)
+
+        for i, player_hand in enumerate(game_state.players):
+            print(f"Player {i + 1}'s hand: {', '.join(str(card) for card in player_hand)}")
+
+        hand_ranks = [
+            await workflow.execute_child_workflow(
```

Comment:
U: What is the benefit of using a child workflow here?

Reply:
It's to figure out the relationship between parent and child workflow.

Reply:
Yup, just because it's possible. This project is about doing what's possible to learn, hence the statements about this being about learning, not about production level code.